Feature/6

YumeAyasaki opened the pull request:
In this PR, I have:

- Checkout from the feature/8 branch...
- Make a footer component...
- Add icons to the public folder...
- Include said component to the _app.js file, thus making it a theme component...
- And IDK if this's needed or not, I make the dropdown menu (Navbar) can be turned off (?) by clicking outside of said dropdown...

Note: The same note as the feature/7, about the icons. And one more, do we need to make this component, like, universal? There's some web that has the Navbar highlight the section they're currently in, but considering our current design, I don't know if we should do that or not...
And I have yet done the linking things...
![localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 12_04_28 PM](https://user-images.githubusercontent.com/43096050/190560597-f338efe6-79f8-42da-86c9-ca3c08fb21ba.png)
![localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 12_04_10 PM](https://user-images.githubusercontent.com/43096050/190560605-ef14329e-ae7d-452f-9d0a-827c9f12a7da.png)
![localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 12_04_19 PM](https://user-images.githubusercontent.com/43096050/190560609-eb3e0200-d3d0-487b-98f5-ee7bb61be3de.png)

YumeAyasaki committed:
Left text (1)

Base (?) for the left text (logo?)...

YumeAyasaki committed:
Navbar for desktop

Basic (schema?) for navbar on desktop...

YumeAyasaki committed:
Prepare for phone

Nothing, just commit to note here...

YumeAyasaki committed:
Add dropdown menu

Add dropdown menu + add other things to make dropdown menu close when click outside. And pointer cursor when hover over button...

YumeAyasaki committed:
Add component to _app.js

... And make it a theme component...

PaulHo0501 commented on components/theme/Navbar.jsx:
```diff
@@ -0,0 +1,58 @@
+        {
+          clicked && 
+          <div className={styles.navBarPhone} ref={dropdown}>
+            <h3 className={styles.title}>Vườn hoa sứ</h3>
```
Change to "Vườn Hoa Sứ"

PaulHo0501 commented on components/theme/Navbar.jsx:
```diff
@@ -0,0 +1,58 @@
+          </div>
+        }
+        {/* Left section aka logo (Page's name, does it count as logo?)*/}
+        <h3 className={styles.left}>
```
Change to "Vườn Hoa Sứ"

PaulHo0501 commented on components/theme/Navbar.jsx:
```diff
@@ -0,0 +1,58 @@
+        <div className={styles.right}>
+            {menuSection.map(section => <h5 key={section.id} className={styles.rightItem}>{section.context}</h5>)}
+            <button type='button' className={styles.rightItemPhone} onClick={e => handleClick(e)}>
+              <Image src='/icons/Bars.svg' alt='button' layout='fill' objectFit='contain'/>
```
Please use [react-icons](https://react-icons.github.io/react-icons/)

PaulHo0501 commented on components/theme/Navbar.module.scss:
```diff
@@ -0,0 +1,88 @@
+
+// Note: 390px
+
+.header {
```
Please make use of mixins and classes

YumeAyasaki committed:
Rename title

Change Vuon hoa su to Vuon Hoa Su...

YumeAyasaki committed:
Add react-icons

Delete the svg file and add react-icons. Note: Not get color from sass file, need more works...

YumeAyasaki committed:
Merge branch 'feature/8' of https://github.com/MEOStudio/vuonhoasu.org into feature/6

YumeAyasaki committed:
Adjusting with global styling

Adjust small thing to make it compatible with new global styling...

YumeAyasaki commented:
Update PR:
- Change to Vuon Hoa Su...
- Apply react-icons...
- Edit minor things to match the new global styling...

Note: The same question from the feature/6 + should I update PR with a new comment like this, or reply to your changes requested comment...?

PaulHo0501 commented:
> 

Either is fine

PaulHo0501 commented on components/theme/Navbar.jsx:
```diff
@@ -0,0 +1,65 @@
+        }
+        {/* Left section aka logo (Page's name, does it count as logo?)*/}
+        <div className={styles.left}>
+          <div className='sub-header-2'>Vườn Hoa Sứ</div>
```
Please rename class after you rename global styling

YumeAyasaki committed:
Merge branch 'feature/8' of https://github.com/MEOStudio/vuonhoasu.org into feature/6

YumeAyasaki committed:
Change name class

Adjust class name to the new global styling...

YumeAyasaki commented:
Update PR:
- Update name class...

PaulHo0501 commented:
Please solve all the conflicts

YumeAyasaki committed:
Merge branch 'develop' into feature/6

PaulHo0501 commented:
LGTM
